chia/data_layer/data_layer.py

```diff
@@ -137,12 +137,14 @@ async def get_root(self, store_id: bytes32) -> Optional[bytes32]:
             self.log.error(f"Failed to get root for {store_id.hex()}")
         return res.node_hash
 
-    async def get_roots(self, store_ids: List[str]) -> List[Optional[bytes32]]:
+    async def get_roots(self, store_ids: List[str]) -> List[Dict[str, Any]]:
         roots = []
         for id in store_ids:
-            res = await self.data_store.get_tree_root(tree_id=bytes32(hexstr_to_bytes(id)))
+            id_bytes = bytes32(hexstr_to_bytes(id))
+            res = await self.data_store.get_tree_root(tree_id=id_bytes)
             if res is None:
-                self.log.error(f"Failed to get root for {id}")
+                self.log.warning(f"Failed to get root for {id}")
+                roots.append({"id": id_bytes, "root": None})
                 continue
-            roots.append(res.node_hash)
+            roots.append({"id": id_bytes, "root": res.node_hash})
```

Comment:
We've got the terms 'store id' and 'root hash' and we took different parts from each.  Maybe `"store"` and `"root"` or `"id"` and `"hash"`?  Or just always the full term like below.

Reply:
id, hash